docs/COMMON_ACTIONS.md

```diff
@@ -0,0 +1,20 @@
+# SDS Commmon Actions
+
+These samples showcase some of the commonly performed actions in SDS. These include creating, updating, deleting, and reading OCS types and streams.
+
+## [WaveForm](https://github.com/osisoft/OSI-Samples-OCS/blob/main/docs/SDS_WAVEFORM_README.md)
+
+This example highlights using OCS with an integer based key. All of the principals and actions shown here apply to time-series keyed streams as well. Please see the
+[WaveForm](https://github.com/osisoft/OSI-Samples-OCS/blob/main/docs/SDS_WAVEFORM_README.md) readme for a complete listing of operations shown.
+
+## [Time-Series](https://github.com/osisoft/OSI-Samples-OCS/blob/main/docs/SDS_TIME_SERIES_README.md)
+
+This example highlights using OCS with a time-series based key. All of the principals and actions shown here apply to time-series keyed streams as well. PLease see the [Time-Series](https://github.com/osisoft/OSI-Samples-OCS/blob/main/docs/SDS_TIME_SERIES_README.md) readme for a complete listing of operations shown.
```

Comment:
should we say `index` instead of `key`? The only place I see `key` used is the `IsKey` attribute, but the public docs from what I can tell only use `index`, but I might be missing it